Graphs/LineGraph/index.js

```diff
@@ -454,7 +450,7 @@ class LineGraph extends XYGraph {
           .tickSizeOuter(yTickSizeOuter);
 
         if (yTickFormat || yTickFormat === "") {
-            const yAxisTickFormat = (yTickFormatType === duration) ? (d) => moment.duration(d).format(yTickFormat, { trim: false }) : format(yTickFormat);
+            const yAxisTickFormat = (yTickFormatType === duration) ? (d) => formatDate(addMillisecs(new Date(new Date(0).getTimezoneOffset()*1000*60), d), yTickFormat) : format(yTickFormat);
```

Comment:
why all this when you can use date-fns duration? Why did you decide on this?

Reply:
This code formats the provided milliseconds(d) duration as per the format string (yTickFormat ) provided.
There is no equivalent date-fns API for this.
I already explored the below APIs, but found them unsuitable for this,above purpose, hence the round-about approach.
https://date-fns.org/v2.15.0/docs/formatDistanceStrict
https://date-fns.org/v2.15.0/docs/formatDuration